Context/ConfiguredContext.php

```diff
@@ -60,6 +60,9 @@ class ConfiguredContext implements SearchContextInterface
      */
     private $interceptorProvider;
 
+    /** @var ContextSortAttribute */
```

Comment:
Objection creation runtime would be the only reason not to, especially since this is within a loop and this itself in a loop but ok